.github/workflows/ubuntu-workflow.yml

```diff
@@ -36,10 +36,38 @@ jobs:
         with:
           arguments: "clean build"
 
-      - name: Add ubuntu-workflow Build Scan URL to Pull Request
-        uses: mshick/add-pr-comment@v1
+      - name: Get current time
+        uses: 1466587594/get-current-time@v2
+        id: current-time
         with:
-          repo-token: ${{ secrets.GITHUB_TOKEN }}
-          message: |
+          format: 'YYYY-MM-DD HH:mm:ss'
+          utcOffset: "+00:00"
+
+      - name: Find Comment
+        uses: peter-evans/find-comment@v1
+        id: fc
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
```

Comment:
Probably for push events it will be null

Reply:
maybe skip past push events and only check for a creation of pull request? @pawelpasterz